keel-api/src/main/kotlin/com/netflix/spinnaker/keel/plugin/KeelPlugin.kt

```diff
@@ -1,21 +0,0 @@
-/*
- * Copyright 2018 Netflix, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.netflix.spinnaker.keel.plugin
-
-interface KeelPlugin {
-  val name: String
-    get() = javaClass.simpleName
-}
```

Comment:
I think this is literally the only actual change here. The only sub-type of this interface was `ResourceHandler` so I thought it made sense to get rid of it.

Reply:
Hmm... What about #748?

Reply:
I think whether the result of that would require a common top-level interface is an open question